src/components/RoomConnect.vue

```diff
@@ -0,0 +1,65 @@
+<template>
+  <div class="room-connect">
+    <input type="text" v-model="roomId" placeholder="Enter room id ...">
+    <button @click="joinRoom"> Join Room </button>
+    <button @click="createRoom"> Create Room </button>
+  </div>
+</template>
+
+<script setup lang="ts">
+import { v4 as uuid } from 'uuid';
+import { ref, defineEmits } from 'vue';
+import type { Ref } from 'vue';
+import { useVideoStore } from "../stores/videoStore";
+import {ClientSocketHandler} from "../modules/socket";
+import { useSocketStore } from '../stores/socketStore';
+import { videoHandler } from '../modules/video';
+import { Socket } from 'socket.io-client';
+
+const videoStore = useVideoStore();
+const socketStore = useSocketStore();
+const emit = defineEmits(['open']);
+let roomId: Ref<string> = ref('');
+
+
+function joinRoom() {
```

Comment:
IMO we should switch to ES6 arrow functions to make it more consistent across the project